docs/source/commands.rst

```diff
@@ -67,6 +68,34 @@ The ``agent`` command will run a Beamer agent and provide liquidity for the brid
        takes precedence for this chain. Default: ``5.0``.
 
 
+.. _command-config-read:
+
+``beamer config read``
+^^^^^^^^^^^^^^^^^^^^^^
+
+``beamer config read --rpc-file RPC-FILE --abi-dir DIR --artifact CHAIN-NAME.deployment.json STATE_PATH``
+
+The command reads the latest contract configuration state from the chain and
+store it into ``STATE_PATH``. If ``STATE_PATH`` already exists, it is used as
+the starting point to fetch contract events from. Otherwise, contracts events
+are fetched from the deployment block.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Command-line option
+     - Description
+
+   * - ``--abi-dir DIR``
+     - The directory containing contract ABI files.
+
+   * - ``--artifact CHAIN-NAME.deployment.json``
+     - Path to the deployment artifact.
+
+   * - ``--rpc-file``
+     - Path to the JSON file containing RPC information.
+
+
```

Comment:
It would be nice to see an example output.

Reply:
You mean, the example for an RPC file?
If so, I was meaning to have a separate subsection on that and link to it from here and also from the deployment commands, when I get to their docs.